src/index.ts

```diff
@@ -33,7 +33,15 @@ export default class Verto {
   public arweave!: Arweave;
   public keyfile!: JWKInterface | undefined;
 
-  constructor(keyfile?: JWKInterface, arweave?: Arweave) {
+  public exchangeContract!: string;
+  public exchangeWallet!: string;
+
+  constructor(
+    keyfile?: JWKInterface,
+    arweave?: Arweave,
+    exchangeContract?: string,
+    exchangeWallet?: string
```

Comment:
Why not have an interface called Options and have these fields inside it? It would be more clean :)

Reply:
Brilliant idea. I second.

Reply:
Done